llvm/lib/Target/RISCV/RISCVISelLowering.cpp

```diff
@@ -731,7 +731,7 @@ RISCVTargetLowering::RISCVTargetLowering(const TargetMachine &TM,
                          VT, Custom);
       setOperationAction({ISD::FP_TO_SINT_SAT, ISD::FP_TO_UINT_SAT}, VT,
                          Custom);
-
+      setOperationAction({ISD::LRINT, ISD::LLRINT}, VT, Custom);
```

Comment:
Should this be restricted to i32 and i64 vectors?

Reply:
Should we be bailing out on non-i32/non-i64 vectors in Verifier?

Reply:
Verifier change up at #70839.

Reply:
Will develop a patch for this in the morning. Thanks for explaining why the Verifier approach isn't good.